include/poller.php

```diff
@@ -260,13 +268,24 @@ function poller_exec_function($queue, $funcname, $argv) {
 	$a->process_id = uniqid("wrk", true);
 	$a->queue = $queue;
 
+	$up_duration = number_format(microtime(true) - $poller_up_start, 3);
+
 	$funcname($argv, $argc);
 
 	$a->process_id = $old_process_id;
 	unset($a->queue);
 
 	$duration = number_format(microtime(true) - $stamp, 3);
 
+	$poller_up_start = microtime(true);
+
+	/** With these values we can analyze how effective the worker is.
```

Comment:
please use `/*` to start a comment block. (`/**` would start a doxygen docu comment)